src/SqlVirtualMachine/SqlVirtualMachine.sln

```diff
@@ -18,6 +18,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ScenarioTest.ResourceManage
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestFx", "..\..\tools\TestFx\TestFx.csproj", "{BC80A1D0-FFA4-43D9-AA74-799F5CB54B58}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Network", "..\Network\Network\Network.csproj", "{34D07BEC-C897-4DF2-8D0E-7B7F5CB2E415}"
```

Comment:
The `.sln` file should not be modified. At least there shouldn't be references to network and compute

Reply:
@isra-fel I had to add these references because compute and network needs to build before my tests run as my tests depend on compute and network cmdlets.
I saw similar references in several other solution files and hence thought this it the way to provide this dependency.
https://github.com/Azure/azure-powershell/search?q=network.csproj&unscoped_q=network.csproj

Can you please let me know if this is not the way and if there is a better way to specify this dependency.

Reply:
Reasonable, thanks for the explanation.